pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "keystores":
+
+`gopass mounts`
+
+- Mount a shared git "keystore":
+
+`gopass mounts add keystore_name location_git_repo`
+
+- Search interactively using a keyword:
+
+`gopass show keyword`
```

Comment:
Is `keyword` something that the user can change? If so, could you wrap it in `{{` and `}}` please as I've done in the suggestion above? Same with the below example.

Reply:
:+1: